aioseop_class.php

```diff
@@ -3335,6 +3347,34 @@ public function admin_enqueue_scripts( $hook_suffix ) {
 		add_filter( "{$this->prefix}display_settings", array( $this, 'filter_settings' ), 10, 3 );
 		add_filter( "{$this->prefix}display_options", array( $this, 'filter_options' ), 10, 2 );
 
+		// This ensures different JS files are enqueued only at the intended screens. Preventing unnecessary processes.
+		$extra_title_len = 0;
+		switch ( $hook_suffix ) {
+			// Screens `post.php` & `./aioseop_class.php` share the same `count-char.js`.
+			case 'post.php' :
```

Comment:
maybe add post-new.php so that it works for new posts as well

Reply:
Ah, thank you, good catch!